trustchain_design.md

```diff
@@ -116,19 +116,19 @@ It is to be noted that because of the signature delegation scheme, the entity th
 
 ### Forward secrecy
 
-Forward secrecy is, despite being an interesting property, **not **implemented in the *Tanker Core* SDK. The reason is that the *Tanker Core* SDK is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
+Forward secrecy is, despite being an interesting property, **not **implemented in *Tanker Core*. The reason is that *Tanker Core* is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
 
-We consider implementing an optional forward secure scheme tailored for messaging in a future version of the *Tanker Core* SDK.
+We consider implementing an optional forward secure scheme tailored for messaging in a future version of *Tanker Core*.
 
 ## Separation of trust
 
-The *Tanker Core* SDK is based on the separation of trust. Any *user* needs to trust:
+*Tanker Core* is based on the separation of trust. Any *user* needs to trust:
 
 - the *application* to store and distribute encrypted data
 - *Tanker* to distribute blocks
-- *Tanker* and the *application* not to collude
+- *Tanker* and the *application* to not collude
```

Comment:
The previous form was more formal http://learnersdictionary.com/qa/Split-Infinitives